src/main/resources/io/jenkins/plugins/designlibrary/sample.js

```diff
@@ -1,50 +1,60 @@
 document.addEventListener("DOMContentLoaded", () => {
-  function extractLanguageFromClassList(element) {
-    return Array.from(element.classList)
-      .filter(clazz => clazz.startsWith('language-'))
-      .map(clazz => clazz.replace('language-', ''));
-  }
 
   const url = document.querySelector('head').dataset.rooturl
 
   document.querySelectorAll('.sample-remote')
     .forEach(element => {
       const fileName = element.dataset.sample;
       const executable = element.dataset.executable;
-      const componentName = window.location.href.match(/.+design-library\/(.+)$/)[1]
+
+      // On the inputs page the preview markup link adds a hash to the url which breaks the regex extraction
+      const strippedHash = window.location.href.replace('#', '')
+      const componentName = strippedHash.match(/.+design-library\/(.+)$/)[1]
```

Comment:
Minor bug introduced in https://github.com/jenkinsci/design-library-plugin/pull/281 cc @mawinter69

at least locally in hpi:run I can't reproduce it on weekly.ci.jenkins.io

If I click it locally I get a # fragment added to the url and the page scrolled to the top and then if I refresh the page none of the examples load

On weekly it live updates in place though with no issue